src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  // ({let partner = person.sex === 'f' ? 'husband' : 'wife'})
```

Comment:
```suggestion
export const Person = ({ person }) => (
```
remove commented code and destructure person